### DISCUSSION TITLE
Reset password

### DISCUSSION THREAD

**Discussion by clundin** [HASwitchPlate/HASPone#168] [Q&A] [answered]
I changed admin to a different username and added a password. I'm no longer able to login to the haswitchplate. Is there a way to "factory reset"?

I tried the "First-time HASPone setup or reset" automation. It reboots and reset but not to factory so I can start over.

**Comment by aderusha** [comment, marked as answer]
If it's still talking to MQTT, you can [trigger a factory reset](https://github.com/HASwitchPlate/HASPone/wiki/MQTT-Control#command-syntax) by pushing a message (any message, or blank message) to the topic `hasp/plate01/command/factoryreset` where `plate01` is the name of your HASPone device.  If that doesn't work, you might have to physically get access to it and [reflash with the erase option enabled](https://github.com/HASwitchPlate/HASPone/wiki/Flashing-HASPone-to-an-ESP8266).

**Reply by clundin** [reply]
That worked! Thank you
